docs/guides/how_to_guides/creating_and_editing_expectations/how_to_edit_an_expectation_suite_using_a_disposable_notebook.rst

```diff
@@ -15,53 +15,160 @@ To simplify this workflow, the CLI command ``suite edit`` takes a named :ref:`Ex
+
+            Run this cell to get started.
+
+        3. Run (and edit) any Expectations in the "Create & Edit Expectations" section:
+
+            The middle portion of the notebook contains a code cell to reproduce each Expectation in the Suite.
+
+            - If you re-run these cells with no changes, you will simply recreate all of your existing Expectations (without validations, since there is no Batch of data). At the end of your notebook, the Expectation Suite will contain exactly the same Expectations that you started out with.
+            - If you edit and run any of the Expectations, the new parameters will replace the old ones.
+            - If you don't run an Expectation, it will be left out of the Suite.
+            - You are welcome to modify the existing Expectations and/or add new ones using the `ExpectationConfiguration` syntax (see :ref:`How to create a new Expectation Suite without a sample Batch <how_to_guides__creating_and_editing_expectations__how_to_create_a_new_expectation_suite_without_a_sample_batch>`).
+
+            Until you run the final cell in the notebook, any changes will only affect the Expectation Suite in memory. They won't yet be saved for later.
+
+        4. Save your Expectations and review them in Data Docs
+
+            At the bottom of the notebook, there's a cell labeled "Save & Review Your Expectations." It will:
+
+                #. Save your Expectations back to the configured Expectation Store.
+                #. Rebuild Data Docs.
+                #. Open your Data Docs to the ExpectationSuite page for your review.  (Note: since there is no Batch of data in this mode, validations are not run. Nevertheless, when the Batch of data is available, all your expectations will appear intact, with the ability to run validations in the notebook.)
+
+        5. Delete the notebook
+
+            In general, these Jupyter notebooks should not be kept in source control. In almost all cases, it's better to treat the Expectations as the source of truth, and delete the notebook to avoid confusion. (You can always auto-generate another one later.)
+
+            The notebook will be stored in the ``great_expectations/uncommitted`` directory. You can remove it like so:
+
+            .. code-block:: bash
+
+                rm great_expectations/uncommitted/edit_movieratings.ratings.ipynb
+
+    .. tab-container:: tab2
+        :title: Show Docs for V3 (Batch Request) API With a Batch of data
+
+        1. Generate a notebook.
+
+            To edit a suite called movieratings.ratings you could run the CLI command:
+
+            .. code-block:: bash
+
+                great_expectations --v3-api suite edit movieratings.ratings --interactive
+
+            For convenience, the :ref:`Data Docs <reference__core_concepts__datasources>` page for each Expectation Suite has the CLI command syntax for you. Simply press the “How to Edit This Suite” button, and copy/paste the CLI command into your terminal.
```

Comment:
Since we have no logic in the GUI button about v3, this sentence is not correct. We should probably remove it and reinstate it when there are no API switches.

Reply:
Fixed in next commit.  Nice catch!  Thanks!